<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="chkPeed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator { width:35px; height: 35px; }</string>
   </property>
   <property name="text">
    <string>Peed</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>dog-peeing-icon.svg</normaloff>dog-peeing-icon.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkPooped">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator { width:35px; height: 35px; }</string>
   </property>
   <property name="text">
    <string>Pooped</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>dog-pooping-icon.svg</normaloff>dog-pooping-icon.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/Adwaita/scalable/emblems/emblem-ok-symbolic.svg</normaloff>../../../../../usr/share/icons/Adwaita/scalable/emblems/emblem-ok-symbolic.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/Adwaita/scalable/actions/window-close-symbolic.svg</normaloff>../../../../../usr/share/icons/Adwaita/scalable/actions/window-close-symbolic.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dteTarget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>701</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit::up-button { width:90px; height: 60px; }
QDateTimeEdit::down-button { width:90px; height: 60px; }</string>
   </property>
   <property name="displayFormat">
    <string>ddd yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>chkPeed</tabstop>
  <tabstop>chkPooped</tabstop>
  <tabstop>dteTarget</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
